### PR TITLE
MM-13225 Add E2E test for channel display modes

### PR DESCRIPTION
**Pull request by cometkim**
#### Summary
Add new E2E test cases for channel display mode
- [x] Full width
- [x] Fixed width, centered

#### Ticket Link
https://github.com/mattermost/mattermost-server/issues/10033

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)

### CODE REVIEW

**Comment by cometkim** [comment]
To test fixed width in visual, the center channel requires width bigger than 1000 but it has 979, so I can validate only its CSS property. 

Can I adjust the display size to see this spec?

**Comment by cometkim** [comment]
Oh well, I can use `cy.viewport()` API to adjust in scope

**Comment by cometkim** [comment]
There is an issue to handle cookies in cypress. [jump to convo](https://community.mattermost.com/core/pl/fig881m9x7rttm1d4nxfwua9ny)

And this will also fix other tests that have not been verified correctly.

**Comment by jasonblais** [comment]
Woot, thanks @cometkim!! Will defer reviews on this PR to devs who are more familiar with the changes.

Thanks for helping contribute! 👍

File: cypress/support/index.js [hunk 1]
```diff
@@ -6,3 +6,8 @@
 // ***********************************************************
 
 import './commands';
+
+// Add login cookies to whitelist to preserve it
+Cypress.Cookies.defaults({
+    whitelist: ['MMAUTHTOKEN', 'MMUSERID', 'MMCSRF'],
+});
```
Comment (saturninoabril) [inline cypress/support/index.js]: 💯 Awesome, thanks for figuring this out!

File: cypress/integration/account_settings/display/channel_display_mode_spec.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        cy.get('#displayButton').should('be.visible');
+
+        // 2. Click the Display tab
+        cy.get('#displayButton').click();
+
+        // * Check that it changed into the Display section
+        // * Check the min setting view if each element is present and contains expected text values
+        cy.get('#displaySettingsTitle').should('be.visible').should('contain', 'Display Settings');
+        cy.get('#channel_display_modeTitle').should('be.visible').should('contain', 'Channel Display');
+        cy.get('#channel_display_modeDesc').should('be.visible').should('contain', 'Fixed width');
```
Comment (deanwhillier) [inline cypress/integration/account_settings/display/channel_display_mode_spec.js]: Is the default value `Full width` here? My login says `Full width` here, but I can't remember if I changed it during testing over the last couple days.

Are we looking to test defaults?

**Comment by deanwhillier** [comment]
LGTM ... just had the one question above. :)

File: cypress/integration/account_settings/display/channel_display_mode_spec.js (hunk 2 above)
Comment (deanwhillier) [reply cypress/integration/account_settings/display/channel_display_mode_spec.js]: I see the ticket asked for fixed width so my question may not be relevant, but I'm still curious. :)

Comment (cometkim) [reply cypress/integration/account_settings/display/channel_display_mode_spec.js]: Yes the default is full width, but the user-1 in test-data has fixed width in first load

Comment (deanwhillier) [reply cypress/integration/account_settings/display/channel_display_mode_spec.js]: Great, thanks for the clarification!

**Pull request merged**